repository/quattor_externals_arch.pan

```diff
@@ -0,0 +1,15 @@
+# Template allowing to create the repository appropriate for Quattor externals,
+# based on current machine arch/os
+
+structure template repository/quattor_externals_arch;
+
+"name" = 'quattor_externals_arch';
+"owner" = "quattor-grid@lists.sourceforge.net";
+"protocols" = list(
+  nlist("name", "http",
+        "url", format("http://yum.quattor.org/externals/%s/%s",
+                      OS_VERSION_PARAMS['arch'],
+                      OS_VERSION_PARAMS['family']+OS_VERSION_PARAMS['majorversion'])
```

Comment:
Why not make this a third parameter to `format()` instead of using `+`, it seems inconsistent.